lib/resque/pool/cli.rb

```diff
@@ -41,6 +42,8 @@ def parse_options
           opt.on('-e', '--stderr FILE', "Redirect stderr to logfile") { |c| opts[:stderr] = c }
           opt.on('--nosync', "Don't sync logfiles on every write") { opts[:nosync] = true }
           opt.on("-p", '--pidfile FILE', "PID file location") { |c| opts[:pidfile] = c }
```

Comment:
`--pidfile FILE` should also override `--no-pidfile`; `opts.delete(:no_pidfile)`